### ISSUE TITLE
Breaks Automatic Syntax Highlighting for some filetypes

### ISSUE THREAD

**Issue by Treeniks** [Monokai/monokai-pro-sublime-text#173]
I use Sublime Text 4169 on Windows 11 and since one of the recent updates, automatic selection of syntax seems to be broken for both Julia and Scala files (there might be more, but those are the two I have noticed). After going through all my installed Packages, I found Monokai Pro to be the culprit. If I remove or disable the packages, the problem goes away.

For Julia I use the [Julia-sublime Package](https://github.com/JuliaEditorSupport/Julia-sublime).

Specifically, when opening a Julia file, e.g. `hello.jl`, instead of Sublime automatically recognizing that it is a Julia file and setting the syntax to Julia, is sets it to `Plain Text (Julia)` with no Syntax highlighting until I manually set the Syntax to Julia.

For Scala it's a little weirder. When opening a scala file, e.g. `hello.scala`, Sublime correctly recognizes and sets the Syntax to Scala, however there is still not Syntax highlighting. Only once I set the Syntax to Scala again manually do I get syntax highlighting.

Mind you this issue is not dependent on using the Theme. Simply having the package installed is enough for this problem to occur, even when using a different theme.

**Comment by dndrsn** [comment]
This is likely related to / a duplicate of https://github.com/Monokai/monokai-pro-sublime-text/issues/178.

**Comment by LimpEmu** [comment]
For me HTML files are broken, I really need this fixed. It's very, very annoying to set the file type manually for one of the most frequently edited file types. For now I will no longer use Monokai Pro even though I paid for it.
